client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-item.js

```diff
@@ -61,8 +61,10 @@ class LabelItem extends Component {
 
 		return (
 			<span>
-				<ReprintDialog siteId={ siteId } orderId={ orderId } { ...label } />
-				<a href="#" onClick={ openDialog } >{ translate( 'Reprint' ) }</a>
+				<ReprintDialog siteId={ siteId } orderId={ orderId } download={ label.returningLabelIndex != null } { ...label } />
+				<a href="#" onClick={ openDialog } >
+					{ label.returningLabelIndex == null ? translate( 'Reprint' ) : translate( 'Redownload' ) }
```

Comment:
`Redownload` sounds so weird. What about `Download again`, or if that's too long, just `Download`? I'd sacrifice being technically correct for being more compresensible.